models/pull.js

```diff
@@ -28,6 +28,18 @@ function Pull(data, signatures, comments, commitStatus, labels) {
       this.data.closes = data.closes;
       this.data.connects = data.connects;
    }
+
+   var isPassing = function() {
+      var repoConfig = config.repos.find(repo => repo.full_name === this.data.repo);
+      var requiredBuilds = repoConfig.required_passing_builds;
+
+      var passingBuilds = this.commitStatuses.filter(
+         build => build.state === 'success');
+
+      return !!_.without(requiredBuilds, ...passingBuilds);
```

Comment:
There's a bug here. requiredBulds is an array of strings. passingBuilds is an array of commit status objects.

I think this would be better expressed like:
```
required_builds.all(required_build -> 
   passingStatuses.find(status -> status.branch == required_build)
)
```

Reply:
1e9bb376676a6ad3be9ddd32622cc34700e8ed7d